debian/control

```diff
@@ -4,11 +4,16 @@ Priority: extra
 Maintainer: Sipwise Development Team <support@sipwise.com>
 Build-Depends: debhelper (>= 5),
                iptables-dev (>= 1.4),
-               libavcodec-dev,
-               libavfilter-dev,
-               libavformat-dev,
-               libavresample-dev,
-               libavutil-dev,
+               libavcodec-dev (>= 6:10),
+               libavfilter-dev (>= 6:10),
+               libavformat-dev (>= 6:10),
+               libavresample-dev (>= 6:10),
+               libavutil-dev (>= 6:10),
+               libavcodec55 (>= 6:10) | libavcodec56, 
```

Comment:
There are trailing white-spaces in some lines, maybe you could just drop them?
Also would be nice to run `wrap-and-sort -a`  so the changes to Build-Depends are sorted alphabetically.